tasks/bond_configuration.yml

```diff
@@ -16,8 +15,9 @@
 - name: Create the network configuration file for bond devices
   become: true
   template:
-    src: 'bond_{{ ansible_facts.os_family }}.j2'
-    dest: '{{ interfaces_net_path[ansible_facts.os_family|lower] }}/ifcfg-{{ item.device }}'
+    src: "bond_{{ interfaces_use_nmconnection | ternary('nmconnection', ansible_facts.os_family) }}.j2"
+    dest: "{{ interfaces_net_path }}/{{ interfaces_use_nmconnection | ternary(item.device ~ '.nmconnection', 'ifcfg-' ~ item.device) }}"
+    mode: "{{ interfaces_use_nmconnection | ternary('0600', omit) }}"
```

Comment:
NetworkManager wants additional security to permissions on these files.

Since the previous task did not specify mode, I think it defaulted to 644, which was/is fine for network-scripts/ifupdown. (It's what's already deployed many places, and unsure if it would result in bouncing a lot of devices just because file-permissions changed.)

Though it's probably a good idea to tighten the security for all. Let me know what you think.

Reply:
Ok, let's leave it as it is for now then.